SmartCar/Arduino/sketch_main/sketch_main.ino

```diff
@@ -18,17 +22,27 @@
 int flame_pin;
 int velocity;
 int range;
+int tempC;
```

Comment:
Why is the temp variables declared next to the range and velocity? Are they related?